orca-core/src/main/java/com/netflix/spinnaker/orca/config/UserConfiguredUrlRestrictions.java

```diff
@@ -68,13 +74,15 @@ public UserConfiguredUrlRestrictions build() {
   private final boolean rejectLocalhost;
   private final boolean rejectLinkLocal;
   private final Set<String> rejectedIps;
+  private final List<String> whitelistedDomains;
```

Comment:
I prefer `allowedDomains`

Reply:
Could go with `allowedHostnames` to match the `allowedHostnamesRegex`.

I could see `allowedHostnames` just containing a list of strings that may themselves be regexes. This would allow for a simpler replacement of `allowedHostnamesRegex`.